### ISSUE TITLE
How to use a GigE camera

### ISSUE THREAD

**Issue by evstfani** [dusty-nv/jetson-inference#166]
Hello,
I would like to know if there is a way to use a GigE camera instead of the onboard or a USB camera.
Thank you.

**Comment by kanakiyab** [comment]
You can probably use gstreamer rtsp pipeline to access the GigE stream and modify the gstCamera.cpp. Especially, the buildlaunchstr(). Other changes may be required based on your pipeline.

**Comment by raaka1** [comment]
yes you can use, most companies provide SDK. Our company has done it.
